src/main/kotlin/org/gitanimals/guild/domain/GuildService.kt

```diff
@@ -28,6 +28,12 @@ class GuildService(
         autoJoin: Boolean,
         createLeaderRequest: CreateLeaderRequest,
     ): Guild {
+        val forbiddenWords = listOf("fuck")
+
+        require(forbiddenWords.none { title.contains(it, ignoreCase = true) }) {
+            "The title contains forbidden words and cannot be used."
```

Comment:
gitanimals는 내부적으로 elk를 사용하고 있으며, 여기서 응답하는 로그가 kibana와 slack에 보이게 되는데요.
에러 모니터링과 forbiddenWords 필터링 로직 파악측면에서 아래와 같이 수정하면 어떨까요?

마찬가지로 Fat domain 측면에서 이 로직은 도메인 모델에 들어가야 할 것 같습니다.

```suggestion
            "Title contains forbidden word. title : \"$title\""
```